### PR TITLE
Update web-nfc recordType tests

### PR DESCRIPTION
**Pull request by xiuqijix**
part of #14389

### CODE REVIEW

File: web-nfc/resources/nfc_help.js [hunk 1]
```diff
@@ -68,14 +68,13 @@ function assertWebNDEFMessagesEqual(a, b) {
...
-    return navigator.nfc.push(pushedMessage)
-      .then(() => {
-        return new Promise(resolve => {
-          navigator.nfc.watch((message) => resolve(message), watchOptions);
-        }).then((message) => {
-          assertWebNDEFMessagesEqual(message, pushedMessage);
-        });
-      });
+  promise_test(async t => {
+    let writer = new NFCWriter();
```
Comment (Honry) [inline web-nfc/resources/nfc_help.js]: Nit: s/let/const

File: web-nfc/nfc_recordType_empty-manual.https.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset=utf-8>
-<title>Web NFC Test: push and watch empty records</title>
+<title>Web NFC Test: push and read empty records</title>
```
Comment (Honry) [inline web-nfc/nfc_recordType_empty-manual.https.html]: Nit: s/push/write

Comment (kaixinjxq) [reply web-nfc/nfc_recordType_empty-manual.https.html]: @Honry, we need replace `push` with `write` in all descriptions, right?

File: web-nfc/resources/nfc_help.js (hunk 1 above)
Comment (kaixinjxq) [reply web-nfc/resources/nfc_help.js]: The same as above, we need replace all `let` with `const`, or only this one?

File: web-nfc/nfc_recordType_empty-manual.https.html (hunk 2 above)
Comment (Honry) [reply web-nfc/nfc_recordType_empty-manual.https.html]: Right.

File: web-nfc/resources/nfc_help.js (hunk 1 above)
Comment (Honry) [reply web-nfc/resources/nfc_help.js]: All `let` could be replaced with `const`.

**Comment by kaixinjxq** [comment]
@Honry, many thanks for your review! I just submit a new commit to address your comments, PTAL.

File: web-nfc/nfc_recordType_opaque-manual.https.html [hunk 3]
```diff
@@ -21,13 +21,13 @@
 
 setup({ explicit_timeout: true });
 
-let desc = "Test that nfc.push and nfc.watch succeed when recordType is set to 'opaque'."
-let watchOptions = {
+const desc = "Test that write and read data succeed when recordType is set to 'opaque'."
+const watchOptions = {
```
Comment (Honry) [inline web-nfc/nfc_recordType_opaque-manual.https.html]: Nit: s/watchOptions/readOptions
Please update the rest with same name. :)

Comment (kaixinjxq) [reply web-nfc/nfc_recordType_opaque-manual.https.html]: Updated

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**